fabric-entity-events-v1/build.gradle

```diff
@@ -1,10 +1,12 @@
 archivesBaseName = "fabric-entity-events-v1"
-version = getSubprojectVersion(project, "1.1.0")
+version = getSubprojectVersion(project, "1.2.0")
 
 moduleDependencies(project, [
 		'fabric-api-base'
 ])
 
 dependencies {
 	testmodImplementation project(path: ':fabric-command-api-v1', configuration: 'dev')
+	testmodImplementation project(path: ':fabric-networking-api-v1', configuration: 'dev')
+	testmodImplementation project(path: ':fabric-registry-sync-v0', configuration: 'dev')
```

Comment:
How does this impl depend on registry sync? Assuming the networking api is the prerequisite of registry sync.

Reply:
It registers a test block and a test item, which need registry sync on a test dedi server. (And also if running with different combinations of test mods that register stuff to prevent ID mixups.)

Reply:
Oh right this is testmod, my bad, thought it was the api's impl